### ISSUE TITLE
Add $empty modifier

### ISSUE THREAD

**Issue by Alex-302** [AdguardTeam/AdguardBrowserExtension#280]
$empty should return empty 200 OK response.

Related threads:
https://forum.adguard.com/index.php?threads/cnet-com-zdnet-com-applying-empty-fixes-filtration-errors.11336/#post-88995
https://forum.adguard.com/index.php?threads/applying-empty-does-not-fix-filtration-errors-in-ie.11828/

**Comment by seanl-adg** [comment]
We need to take account of this issue https://github.com/AdguardTeam/AdguardForWindows/issues/1360

**Comment by ameshkov** [comment]
I guess it is possible to implement in form of a redirect to data: url

**Comment by seanl-adg** [comment]
It seems to be not working in chrome 2.6.3 beta.
![2017-05-04 2](https://cloud.githubusercontent.com/assets/22858651/25698701/d9ac4c0a-30fa-11e7-98b4-7fb123c8ddcc.png)
Requests to `wabxsybclllz.com` should not fire.

**Comment by Alex-302** [comment]
@seanl-adg it works with manually added `||wabxsybclllz.com^$important,empty`

<details>

![image](http://i89.fastpic.ru/big/2017/0504/23/54a79a964f100f66b38058c962f53523.jpg)
</details>

Which rule should to block that request in your example?

**Comment by seanl-adg** [comment]
`||popads.net/pop.js^$empty,important` should block it.
The script snippet initiating the request:
```

**Comment by atropnikov** [comment]
In the case of using `$empty` modifier, request finishes with status `307 Internal Redirect` (it's chrome API behavior) and in this case, `onerror` callback is fired. I should investigate, can we simulate `200 OK` response.

**Comment by atropnikov** [comment]
Or, be more precisely the following error `Access to Script at 'data:,' from origin 'null' has been blocked by CORS policy: Invalid response. Origin 'null' is therefore not allowed access.` invokes `onerror` callback.

**Comment by seanl-adg** [comment]
An issue https://github.com/AdguardTeam/AdguardForWindows/issues/1360#issuecomment-333133473 pertains to Windows application is present in browser extensions too. It can be reproduced by following the same steps.

**Comment by ameshkov** [comment]
Filed a new issue:
https://github.com/AdguardTeam/AdguardBrowserExtension/issues/879
